packages/engine-ui/src/utils/mapUtils.js

```diff
@@ -211,74 +213,77 @@ export const toGeoJsonLayer = (id, data, callback) =>
     onClick: callback,
   })
 
-export const toBookingIconLayer = (data, activeBookingId) => {
-  if (!data.length) {
-    return
-  }
+const getIconType = (type) => {
```

Comment:
I would rename this variable. Are we really getting what type of icon it is? Or are we rather using the type of something else to decide which icon mapping to return?
```suggestion
const getIconMappingFromEntityType = (type) => {
```